param/__init__.py

```diff
@@ -201,12 +191,13 @@ def guess_param_types(**kwargs):
                 params[k] = Tuple(**kws)
         elif isinstance(v, list):
             params[k] = List(**kws)
-        elif isinstance(v, np.ndarray):
-            params[k] = Array(**kws)
         else:
+            from numpy import ndarray
```

Comment:
This function seems to assume that numpy and pandas can be imported. Seems like that ought to be checked, not assumed?

Reply:
Yes you're right, this utility was assuming that pandas and numpy were always available. See https://github.com/holoviz/param/pull/630

Reply:
Fixed in a separate PR.